docs/cluster_management.md

```diff
@@ -31,3 +31,25 @@ You can use the kube-version-change utility to convert config files between diff
 $ hack/build-go.sh cmd/kube-version-change
 $ _output/go/bin/kube-version-change -i myPod.v1beta1.yaml -o myPod.v1beta3.yaml
 ```
+
+### Upgrading the kernel on a node
```

Comment:
What does this have to do with upgrading a kernel?  (Yes it's a good example, but updating libc would have the same issues)

This is about preparing a node for cluster removal, either temporary or permanent...   I'll try to think of a better title, but saying this is only about kernel isn't good...